sdk/python/tests/integration/online_store/test_python_feature_server.py

```diff
@@ -1,150 +0,0 @@
-import json
-from typing import List
-
-import pytest
-from fastapi.testclient import TestClient
-
-from feast.errors import PushSourceNotFoundException
-from feast.feast_object import FeastObject
-from feast.feature_server import get_app
-from feast.utils import _utc_now
-from tests.integration.feature_repos.repo_configuration import (
-    construct_universal_feature_views,
-)
-from tests.integration.feature_repos.universal.entities import (
-    customer,
-    driver,
-    location,
-)
-
-
-@pytest.mark.integration
-@pytest.mark.universal_online_stores
```

Comment:
This runs the tests for all online stores and basically tests the feature server end to end.

Reply:
it does, but using the fake server via fastapi testclient so not a "true" integration e2e. what ends up happening is that it's basically a unit test of the server and integration test for the feature store. the integration test of the feature store w the diff online stores already happens in [test_universal_online](https://github.com/feast-dev/feast/blob/35fbdc91cc8a327be2a8cd162f7ccff983b16932/sdk/python/tests/integration/online_store/test_universal_online.py). there isn't, however, a test suite dedicated to working w the server, only that implicit one baked into this file. 

since this construct is proving to be complicated and brittle, seemed much simpler to break the implicit unit test out from the integration test and let the two exist on their own. imo if we really wanted a full e2e we'd start up a server and make calls to it w a regular client, not the fastapi one. reasonable people can def disagree on this file, but figured worth considering this option since integration tests have been failing inconsistently.

Reply:
Cool, I can get behind that.